packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.test.tsx

```diff
@@ -194,7 +194,13 @@ describe('QueryWrapper', () => {
       await waitFor(() => expect(providedContext).toBeDefined())
 
       act(() => {
-        providedContext!.executeQueryRequest(initialQueryRequest)
+        providedContext!.executeQueryRequest({
+          ...initialQueryRequest,
+          query: {
+            ...initialQueryRequest.query,
+            offset: 10,
+          },
+        })
```

Comment:
Previously tested `onQueryChange`, but the query didn't actually change. Update the test to actually change the query. Should we define/test the behavior for the edge case where the query was updated but not changed?

Reply:
It's not the worst thing if the query is executed when the query is updated but unchanged, so I would not prioritize this unit test.